phys2bids/tests/test_integration.py

```diff
@@ -73,6 +73,9 @@ def test_integration_acq(skip_integration, samefreq_full_acq_file):
     assert json_data['Columns'] == ['time', 'RESP - RSP100C', 'PULSE - Custom, DA100C',
                                     'MR TRIGGER - Custom, HLT100C - A 5', 'PPG100C', 'CO2', 'O2']
 
+    shutil.copy(join(test_path, 'phys2bids_report.html'), '/tmp/src/phys2bids/reporting')
+    shutil.copy(join(test_path, 'phys2bids_report_log.html'), '/tmp/src/phys2bids/reporting')
```

Comment:
I think these are copying to a file named `reporting`, not copying both files to a new `reporting` folder. It's possible that just adding a trailing `/` will fix it.

```suggestion
    shutil.copy(join(test_path, 'phys2bids_report.html'), '/tmp/src/phys2bids/reporting/')
    shutil.copy(join(test_path, 'phys2bids_report_log.html'), '/tmp/src/phys2bids/reporting/')
```

Reply:
Okay, let's try it ou!